### PR TITLE
Fixed all eslint warnings

### PR DESCRIPTION
**Pull request by usmanajmal**
Warnings included
- Missing semicoln warnings
- Line exceeding 120 characters

Signed-off-by: Usman Ajmal <syedusmanajmal@gmail.com>

### CODE REVIEW

File: src/renderers/dom/fiber/ReactDOMFiber.js [hunk 1]
```diff
@@ -55,7 +55,10 @@ var DOMRenderer = ReactFiberReconciler({
     return domElement;
   },
 
-  prepareUpdate(domElement : Instance, oldProps : Props, newProps : Props, children : HostChildren<Instance>) : boolean {
+  prepareUpdate(domElement : Instance,
```
Comment (zpao) [inline src/renderers/dom/fiber/ReactDOMFiber.js]: Please put each argument on its own line indented 2 space, closing paren with bracket on last line unindented

```js
fn(
  arg,
  arg
) {
```

File: src/renderers/noop/ReactNoop.js [hunk 2]
```diff
@@ -169,7 +169,8 @@ var ReactNoop = {
     }
 
     function logFiber(fiber : Fiber, depth) {
-      console.log('  '.repeat(depth) + '- ' + (fiber.type ? fiber.type.name || fiber.type : '[root]'), '[' + fiber.pendingWorkPriority + (fiber.pendingProps ? '*' : '') + ']');
+      console.log('  '.repeat(depth) + '- ' + (fiber.type ? fiber.type.name || fiber.type : '[root]'),
```
Comment (zpao) [inline src/renderers/noop/ReactNoop.js]: Same goes for calls.

**Comment by zpao** [comment]
I like the idea and we'll take this if you update to match our style. Thanks :)

File: src/renderers/dom/fiber/ReactDOMFiber.js (hunk 1 above)
Comment (usmanajmal) [inline src/renderers/dom/fiber/ReactDOMFiber.js]: That's a better style. I will push a patch. :)

**Comment by usmanajmal** [comment]
Couldn't figure out a way to push patch here to this issue so created a new pull request: https://github.com/facebook/react/pull/7247. 

Hope that's acceptable. :)

**Comment by zpao** [comment]
You should be able to just add more commits and push to the same branch. If you can fix that I'd prefer it since your new PR is also going to need more changes and creating a new PR for every followup isn't maintainable.

**Comment by usmanajmal** [comment]
@zpao Yes, sorry about that. I have closed the new non-necessary PR and will update this one in a while.

**Comment by usmanajmal** [comment]
Not sure if I did the correct thing here but I had merge my changes with remote so I did following:

1. git pull origin eslint-fixes
2. Resolved conflicts
3. Pushed without sqashing on suggestion of a reviewer at #7247 (closed)

Please advise if this isn't acceptable. I will modify the commit.

**Comment by zpao** [comment]
Thanks!

Since we can squash & merge this should be ok. Otherwise you would need to rebase first to remove all of the extraneous commits.

File: src/renderers/shared/fiber/ReactChildFiber.js [hunk 3]
```diff
@@ -29,7 +29,12 @@ var {
 var ReactFiber = require('ReactFiber');
 var ReactReifiedYield = require('ReactReifiedYield');
 
-function createSubsequentChild(returnFiber : Fiber, existingChild : ?Fiber, previousSibling : Fiber, newChildren, priority : PriorityLevel) : Fiber {
+function createSubsequentChild(
+  returnFiber : Fiber, 
+  existingChild : ?Fiber, 
+  previousSibling : Fiber, 
+  newChildren, priority : PriorityLevel
```
Comment (zpao) [inline src/renderers/shared/fiber/ReactChildFiber.js]: Can you move `priority` to a new line

Comment (usmanajmal) [inline src/renderers/shared/fiber/ReactChildFiber.js]: sure :)

**Comment by zpao** [comment]
Thanks!

**Pull request merged**